kaniko/executor_kubernetes.go

```diff
@@ -28,69 +28,69 @@ type JobConfig struct {
 
 	// Advanced customisations (raw YAML overrides)
 	ContainerOverride   string // YAML string to override the Kaniko container object.
-	PodTemplateOverride string // YAML string to override the job.spec.template object.
+	PodTemplateOverride string // YAML string to override the pod.spec object.
```

Comment:
To be consistent we should rename to `PodOverride`. A Pod template is referring to the `spec.template` field of a higher order resource like `Job` or `Deployment`.

```suggestion
	PodOverride string // YAML string to override the pod object.
```